pkg/controller/node/cidr_set_test.go

```diff
@@ -334,3 +481,68 @@ func TestOccupy(t *testing.T) {
 		}
 	}
 }
+
+func TestCIDRSetv6(t *testing.T) {
```

Comment:
Same here...add a test name field so user can correlate failure to test case. May want to show expected value too.